projects/storefrontlib/src/cms-structure/outlet/outlet-renderer.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { OutletDirective } from './outlet.directive';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class OutletRendererService {
+  private outletRefs = new Map<string, OutletDirective>();
+
+  render(outlet: string): void {
+    if (this.outletRefs.size !== 0) {
+      this.outletRefs.get(outlet).render();
+    }
+  }
+
+  registerOutlet(cxOutlet: string, context: OutletDirective): void {
```

Comment:
just register?

Reply:
I like it that you leave out the access modifier as public is the default

Reply:
My bad I forgot